docs/content/configuration/creating_pvc.md

```diff
@@ -4,9 +4,11 @@ Create a PersistentVolumeClaim (PVC) YAML file for a persistent volume (PV).
 
 The IBM® block storage CSI driver supports using both file system and raw block volume modes.
 
-**Important:** If not defined, the default mode is `Filesystem`. Be sure to define the mode as `Block` if this configuration is preferred.
+**Important:**
+  - If not defined, the default mode is `Filesystem`. Be sure to define the mode as `Block` if this configuration is preferred.
+  - Only use the `volumegroup` label if the PVC is not using a StorageClass with a defined `volume_group`. The PVC `volumegroup` label value is the key defined in the VolumeGroup YAML.
```

Comment:
important notice - the key and value for the volume group labels are not pre-defined, they need to match the selector in the target volume group (spec.source.selector)

Reply:
so `volumegroup` is not necessary the "label"

Reply:
i called it "label" because that is what it's defined as in the pvc
```

 metadata:
      name: demo-pvc-raw-block
      labels:
        volumegroup: demo-volumegroup
    spec:
```

i will try to update the text better though

Reply:
added info but please check. we haven't mentioned the spec.source.selector before so I don't know if this should be here. thanks